src/pages/products/sgid/recreation/trails-pathways.astro

```diff
@@ -0,0 +1,131 @@
+  // delete this if it is unused
+  hub: {
+    ...dataPages[metadata.title],
+  },
+
+  // links are optional and only used when a product has download type of links that aren't ArcGIS Hub
+  // delete this if it is unused
+  links: [
+    {
+      title: 'The link title',
+      url: 'https://link-to-the-resource.com',
+    },
+  ],
+};
+---
+
+<Layout title={page.title} subTitle={page.updateHistory[0]} category={page.category} actionUrl={page.application}>
+  <Metadata slot="metadata" {...page} />
+
+  <Fragment slot="summary">
+    <p>
+      The Utah Trails and Pathways map layer represents off-street features and important road connections that augment Utah’s recreational trail and transportation network. This map layer features many of Utah’s trails and pathways but is not yet complete.
+    </p>
+  </Fragment>
+
+  <!-- the title for hub data is typically `Use the data` but it can be anything
+  When using direct downloads for apps or services, use `Get the app`, or `Use the service` -->
+  <Section title="Use the data" slot="downloads" titlePosition="center">
+    <!-- a hub download uses the hub component. Only keep one element and delete the unused. -->
+    <HubDownloads {...page.hub} />
+  </Section>
+
+  <Fragment slot="description">
+    <p>This dataset primarily depicts features used by hikers/pedestrians and bikers. It began with map data purchased from a private entity (Orbital View, Inc), for which full rights were purchased by the GOED Office of Outdoor Recreation in July 2014. Since the original purchase, subsequent substantive updates have included: 2014 Revisions from National Park service; 2015 additions from Emery County; June 2016 additions from Cache County US Forest Service Geodata Hub (all forests except Sawtooth in NW Box Elder County), and Mountainland Association of Governments (Utah and Wasatch counties); 2018/2019 additions as part of the multimodal project; and currently ongoing updates by WFRC transportation planners.</p>
+
+    <p>Presently, the planned approach for on street bicycle lanes and routes is to incorporate them in the the statewide roads GIS layer using left and right side attributes. On street bike infrastructure is not expected to be represented in this map layer.</p>
+      
+    <p>Although it is a work in progress, this dataset represents a substantial starting point to integrate additional trail systems from other local, state, and federal agencies. The success of this effort will rely heavily on local government and land stewards to review and continue to improve upon this data. Additional segments can be submitted to UGRC for inclusion in this dataset.</p>
+      
+    <p>Note: Significant enhancements (including a schema change) were made to this layer in June 2020.</p>
+
+      <b>CartoCode</b>
+      <p>For a more specific query of facility types, use the CartoCode field:</p>
+      <li>
```

Comment:
Li's belong inside ol's

Reply:
nice, yeah that fixed the mysterious, superfluous bullet.